### ISSUE TITLE
Visually appearing heading 'NPS Extension for Azure MFA' text is not defined as a heading.

### ISSUE THREAD

**Issue by msftedad** [wixtoolset/issues#9208]
### WiX Version

5.0.0

### .NET or MSBuild or Visual Studio Version

Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.14.20

### HeatWave Version

1.0.6

### Repro Steps

**Pre-requisites:**
1. Download NPS Extension for Azure MFA from here https://www.microsoft.com/en-us/download/details.aspx?id=54688 (Version 1.2.2893.1) and open it.

**Repro Steps:**

1. Open 'NPS Extension for Azure MFA' app in windows.
2. 'NPS Extension for Azure MFA' screen will get displayed on the screen.
3. Turn on Narrator and navigate using 'H' key in heading mode.
4. Verify whether visually appearing heading 'NPS Extension for Azure MFA' text is not defined as a heading or not.

### Actual Result

Visually appearing heading 'NPS Extension for Azure MFA' text is not defined as a heading.

**Refer Attachment:**

1. Visually appearing heading 'NPS Extension for Azure MFA' text is not defined as a heading.png
2. Visually appearing heading 'NPS Extension for Azure MFA' text is not defined as a heading.mp4

https://github.com/user-attachments/assets/22edbc90-9fb0-4840-be16-726903724231
<img width="1330" height="577" alt="Image" src="https://github.com/user-attachments/assets/8cec8e4b-f8fd-454f-8d05-06d182864593" />

### Expected Result

Visually appearing heading 'NPS Extension for Azure MFA' text should be defined as a heading level 1.

### Acknowledgements

- [x] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be [sent to Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions)).
- [x] I am paying the [Open Source Maintenance Fee](https://opensourcemaintenancefee.org/) by [sponsoring the `wixtoolset` project](https://github.com/sponsors/wixtoolset) because I support the maintainers.

**Comment by barnson** [comment]
https://github.com/wixtoolset/issues/issues/6222
